tools/harness-thci/OpenThread.py

```diff
@@ -27,8 +27,8 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """
 >> Thread Host Controller Interface
->> Device : OpenThread THCI
->> Class : OpenThread
+>> Device : nRF_Connect_SDK THCI
```

Comment:
In https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thread_certification.html we recommend to rename from `OpenThread` to `nRF_Connect_SDK`. In order to avoid confusion, and because this will be the regular case, I would keep `OpenThread`.